apps/src/storage/dataBrowser/dataUtils.js

```diff
@@ -15,6 +15,22 @@ export function getDatasetInfo(tableName) {
   return tables.find(table => table.name === tableName);
 }
 
+export function isBlank(value) {
+  return value === undefined || value === '' || value === null;
+}
+
+export function ignoreMissingValues(records, columns) {
+  if (records && columns) {
+    let filteredRecords = records;
+    columns.forEach(column => {
+      filteredRecords = filteredRecords.filter(
+        record => !isBlank(record[column])
+      );
+    });
+    return filteredRecords;
```

Comment:
Nit (although I don't think we've configured our linter to forbid this): We're leaning on the implicit `return undefined` fall-through behavior here, and haven't documented it as such.  It seems like an empty array might be a better default case.